README.md

```diff
@@ -1,16 +1,165 @@
-# nodecloud-cli
+# Nodecloud-cli
```

Comment:
Title should be same as repo name (nodecloud-cli)

Reply:
Changed the name to the repo name